{"name": "Проверка авторизации с несуществующей парой логин-пароль", "status": "passed", "steps": [{"name": "Система возвращает ошибку, если неправильно указать логин или пароль", "status": "passed", "start": 1714888302441, "stop": 1714888302540}, {"name": "Проверяем статус код и тело после авторзиции c несущ.данными", "status": "passed", "start": 1714888302540, "stop": 1714888302540}], "start": 1714888302441, "stop": 1714888302541, "uuid": "39b2e187-3d31-434a-acc3-8c1c8c73400c", "historyId": "c1c0324ec0f6c82f3af86ad77f6b003d", "testCaseId": "c1c0324ec0f6c82f3af86ad77f6b003d", "fullName": "tests.test_login_courier.TestLogin#test_login_with_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "ZALMAN"}, {"name": "thread", "value": "12659-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}